apps/sms/views/inbox/style/edit-mode.css

```diff
@@ -17,24 +17,20 @@
   height: calc(100% - 10rem);
 }
 
-.edit [data-type="list"] li > a > aside.icon-unread {
+html[dir="ltr"] .edit [data-type="list"] li > a > aside.icon-unread {
   transform: translateX(5rem);
 }
+html[dir="rtl"] .edit [data-type="list"] li > a > aside.icon-unread {
+  transform: translateX(-5rem);
+}
 
 .edit #threads-container aside.pack-end {
-  -moz-margin-start: 4rem;
+  margin-inline-start: 4rem;
 }
 
-.edit #threads-container[data-type="list"] aside.pack-end span {
+html[dir="ltr"] .edit #threads-container[data-type="list"] aside.pack-end span {
```

Comment:
Oh can't I just throw `.edit` out? As #threads-container is an id, it should be unnecessary right?